src/main/kotlin/org/wfanet/panelmatch/client/eventpreprocessing/JniPreprocessEvents.kt

```diff
@@ -14,38 +14,29 @@
 
 package org.wfanet.panelmatch.client.eventpreprocessing
 
-import java.lang.RuntimeException
 import java.nio.file.Paths
 import org.wfanet.panelmatch.common.loadLibrary
+import org.wfanet.panelmatch.common.wrapJniException
 import wfanet.panelmatch.client.PreprocessEventsRequest
 import wfanet.panelmatch.client.PreprocessEventsResponse
 
 /** A [PreprocessEvents] implementation using the JNI [PreprocessEvents]. */
 class JniPreprocessEvents : PreprocessEvents {
-  /** Indicates something went wrong in C++. */
-  class JniException(cause: Throwable) : RuntimeException(cause)
 
-  private fun <T> wrapJniException(block: () -> T): T {
-    return try {
-      block()
-    } catch (e: RuntimeException) {
-      throw JniException(e)
+  override fun preprocess(request: PreprocessEventsRequest): PreprocessEventsResponse {
+    return wrapJniException {
+      PreprocessEventsResponse.parseFrom(
+        EventPreprocessing.preprocessEventsWrapper(request.toByteArray())
+      )
     }
   }
 
-  override fun PreprocessEvents(request: PreprocessEventsRequest): PreprocessEventsResponse {
-    return PreprocessEventsResponse.parseFrom((PreprocessEvents(request)).toByteString())
-  }
-
   companion object {
+
     init {
-      loadLibrary(
-        name = "preprocess_events",
-        directoryPath =
-          Paths.get(
-            "panel_exchange_client/src/main/swig/wfanet/panelmatch/client/eventpreprocessing"
-          )
-      )
+      val swigPath =
```

Comment:
I'm afraid I don't know Kotlin, so my comments may not be accurate, but the following stackoverflow discussion seems to suggest that this is not the optimal way to create a constant. 

https://stackoverflow.com/questions/44038721/constants-in-kotlin-whats-a-recommended-way-to-create-them